Makefile

```diff
@@ -166,7 +166,7 @@ codegen: kops-gobindata
 
 .PHONY: protobuf
 protobuf:
-	cd ${GOPATH_1ST}/src; protoc --gofast_out=. k8s.io/kops/protokube/pkg/gossip/mesh/mesh.proto
```

Comment:
could we have this run `./hack/update-header.sh` afterwards, rather than ignoring the file in `hack/verify-boilerplate.sh` ?

and maybe something similar with gofmt below

Reply:
I don't think there is a huge value in having the copyright in this file.
In general, would be best to leave this as untouched as possible including by gofmt. In the future, it will be easier to understand the changes. I am tempted to make a PR for gogo/protobuf and fix the gofmt issue there.

Reply:
https://github.com/gogo/protobuf/pull/702